crates/astria-sequencer-relayer/local.env.example

```diff
@@ -0,0 +1,45 @@
+# Address of sequencer/cometbft/tendermint to request new blocks.
+# 127.0.0.1:26657 is the default socket address at which cometbft
+# serves RPCs.
+ASTRIA_SEQUENCER_RELAYER_SEQUENCER_ENDPOINT="http://127.0.0.1:26657"
+
+# Address of the celestia node. 127.0.0.1:26658 is the default socket
+# address at which celestia node servces RPCs.
+ASTRIA_SEQUENCER_RELAYER_CELESTIA_ENDPOINT="http://127.0.0.1:26658"
+
+# The bearer token to submit sequencer blocks as blobs to Celestia.
+# The token is obtained by running `celestia bridge auth <permissions>`
+# on the host running the celestia node.
+ASTRIA_SEQUENCER_RELAYER_CELESTIA_BEARER_TOKEN="<JWT Bearer token>"
+
+# The gas limit when paying for submitting blobs to celestia
+ASTRIA_SEQUENCER_RELAYER_GAS_LIMIT=1000000
+
+# Whether to disable writing to celestia
+ASTRIA_SEQUENCER_RELAYER_DISABLE_WRITING=false
+
+# The duration in milliseconds that sequencer-relayer sleeps before
+# requesting a new block from sequencer.
+ASTRIA_SEQUENCER_RELAYER_BLOCK_TIME=1000
+
+# The path to the file storing the cometbft validator key.
+# Used to sign data before submitting it to celestia.
+ASTRIA_SEQUENCER_RELAYER_VALIDATOR_KEY_FILE=.cometbft/config/priv_validator_key.json
+
+# The port that sequencer relayer will bind on 127.0.0.1 to serve RPCs.
+ASTRIA_SEQUENCER_RELAYER_RPC_PORT=2450
+
+# The port that sequencer relayer will bind to join the gossipnet p2p network.
+ASTRIA_SEQUENCER_RELAYER_P2P_PORT=33900
+
+# A list of bootnodes that sequencer relayer will use to join the gossipnet p2p network.
+# Optional: if left empty sequencer relayer will be a single node in the gossipnet,
+# waiting for peers to join it.
+ASTRIA_SEQUENCER_RELAYER_BOOTNODES=/ip4/1.2.3.4/tcp/1234
+
+# The path to the libp2p private key.
+# Optional: if left empty sequencer-relayer will only join permissionless gossipnets.
```

Comment:
you can remove this line, the private key is used as authentication on the network (ie. its peer ID, message authentication), not to do with what network it joins